webui/react/src/services/api.ts

```diff
@@ -83,9 +83,15 @@ export const resetUserSetting = generateDetApi<EmptyParams, Api.V1ResetUserSetti
 export const getUserPermissions = generateDetApi<
   Service.GetUserParams,
   Api.V1GetPermissionsSummaryResponse,
-  Type.Permission[]
+  Type.PermissionsSummary
```

Comment:
Was this type just wrong before?

Reply:
Okay I see we were throwing out information in the API response parsing step and now we're not.